collab-database/src/entity.rs

```diff
@@ -418,7 +419,7 @@ impl From<usize> for SelectOptionColor {
   }
 }
 
-#[derive(Debug, Clone, Default, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Debug, Clone, Default, PartialEq, Eq, Serialize_repr, Deserialize_repr)]
```

Comment:
It's not a new data type, won't it break the serialisation with existing clients?

Reply:
It's never been used in previous releases.